LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionProxy/Provider/TransparentProxyProvider.swift

```diff
@@ -130,6 +130,9 @@ open class TransparentProxyProvider: NETransparentProxyProvider {
                 Task {
                     try await self.updateNetworkSettings()
                 }
+            case .proxyAvailable:
```

Comment:
This is added here because of how settings are propagated through our processes but thre's nothin to be done in the provider itself since this won't change a runtime.